tests/std/tests/P0067R5_charconv/test.cpp

```diff
@@ -197,9 +197,11 @@ void test_common_to_chars(
 }
 

```

Comment:
I guess we can avoid using internal mechanisms in test file by replacing them with the following... (no change requested)

```suggestion

#if _HAS_CXX23 // testing constexpr overloads for integral types in C++23 (P2291R3)
#define CONSTEXPR23 constexpr

constexpr is_runtime_evaluated() noexcept {
    return !is_constant_evaluated();
}
#else // ^^^ _HAS_CXX23 / vvv !_HAS_CXX23
#define CONSTEXPR23 inline

constexpr is_runtime_evaluated() noexcept {
    return true;
}
#endif // _HAS_CXX23
```

Reply:
Yes, we could do something like that in the future, possibly centralized in a lightweight classic header.

For the time being, tests don't need to strenuously avoid internal machinery and macros as long as there is reasonable justification.